language-design.md

```diff
@@ -17,6 +17,97 @@ For each quote, describe what it was about the quote that led you pick it.
+for some of the more abstruse syntax,” which can make writing and reading code more intuitive, 
+can also subject the coding language to being deemed only for novices, who need the syntactical 
+hand-holding to understand grammar that experienced coders have programmed in for years. 
+Unfortunately, introducing more “human-friendly” programming languages may subject a language and 
+its users to some of the biases that the Yang and Rabkin article mentioned, and further “set up 
+a hierarchy among programmers that systematically privileges certain groups” [Yang and Rabkin 2015]. 
+In the same way that “computer science PhDs explain they were embarrassed to know Python 
+‘because it’s a language for idiots’ and that C++ is seen, by some as a horrible language, 
+in part because ‘a lot of substandard programmers use it’”, a more user-friendly programming 
+language might not be popular with a wide range of audiences. Given existing biases towards 
+certain language, it would not be surprising if a more user-friendly and intuitive language that is 
+developed may not be immediately be conceived of as legitimate, thus decreasing the job prospects 
+of those who code with them.
+
+--
+
+*“These preconceived biases arise because programming languages are as much social constructs 
+as they are technical ones. A programming language, like a spoken language, is defined 
+not just by syntax and semantics, but also by the people who use it and what they have 
+written. Research shows that the community and libraries, rather than the technical features, 
+are most important in determining the languages people choose” [Yang and Rabkin, 2015].*
+
+It’s interesting to consider how dynamic a language can be. We often think about a language 
+like English as a static medium in that what we consider to be a ‘word’ must currently be 
+defined in some official dictionary, and we forget how so many new ‘words’ are added to the 
+official vocabulary of English every year and even the grammar rules we follow can change through 
+time. This is the result of how people begin to write and speak in different ways depending on 
+their cultural context, and sometimes, different dialects and possibly even new languages can emerge. 
+In the same vein, programming languages are not confined to the rules that were established by 
+their initial founders - in fact, a programming language may evolve so much over time so that it 
+could no longer be credited to the few individuals that ‘created’ it. Rather, a programming language 
+becomes the creation of the entire community of programmers that have written in the language, 
+created useful tools with it, and shared their experiences and solutions with others. This is 
+more in line with the ideas that are suggested by Steele in his talk, Growing a Language.
+
+However, in the context of the article, this quote suggests that these communities also have 
```

Comment:
Agreed! I wonder if perhaps documentation can also be an element which perpetuates this bias.  When I was first learning to program, I remember that reading documentation was a huge struggle.  Just determining what the documentation means (since it would be so often defined in unfriendly and undefined terms) made certain languages less welcoming to newcomers and leads to an air of superiority for those who have already mastered them.